### ISSUE TITLE
Minor clarification on event handler registration comments

### ISSUE THREAD

**Issue by patrickhlauke** [metafizzy/flickity#722]
On https://github.com/metafizzy/flickity/blob/master/dist/flickity.pkgd.js#L2133

>`// Pointer Events. Chrome 55, IE11, Edge 14`

Note that PEs are also supported in practically all Blink-based browsers (so Samsung Internet, Opera, Brave) and hot off the press, Firefox 59+

https://github.com/metafizzy/flickity/blob/master/dist/flickity.pkgd.js#L2136

> ` // listen for both, for devices like Chrome Pixel`

Chromebook Pixel also supports pointer events. Perhaps just mention `listen for both, for older browsers that don't support pointer events but run on devices with both mouse/trackpad and touchscreen` (admittedly a bit wordy)

**Comment by desandro** [comment]
Hi! Thanks for these. Yeah. This comments  and code are now a bit outdated. This code in particular, comes from a dependency package: unipointer. I've opened metafizzy/unipointer#5 to track it over there.
